src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -0,0 +1,930 @@
+	data-wp-text="state.navigation.message"
+></div>
+HTML;
+			};
+
+			add_action( 'wp_footer', $callback );
+		}
+	}
+
+	/**
+	 * Processes the `data-wp-each` directive.
+	 *
+	 * This directive gets an array passed as reference and iterates over it
+	 * generating new content for each item based on the inner markup of the
+	 * `template` tag.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param WP_Interactivity_API_Directives_Processor $p               The directives processor instance.
+	 * @param array                                     $context_stack   The reference to the context stack.
+	 * @param array                                     $namespace_stack The reference to the store namespace stack.
+	 * @param array                                     $tag_stack       The reference to the tag stack.
+	 */
+	private function data_wp_each_processor( WP_Interactivity_API_Directives_Processor $p, array &$context_stack, array &$namespace_stack, array &$tag_stack ) {
+		if ( ! $p->is_tag_closer() && 'TEMPLATE' === $p->get_tag() ) {
+			$attribute_name   = $p->get_attribute_names_with_prefix( 'data-wp-each' )[0];
+			$extracted_suffix = $this->extract_prefix_and_suffix( $attribute_name );
+			$item_name        = isset( $extracted_suffix[1] ) ? $this->kebab_to_camel_case( $extracted_suffix[1] ) : 'item';
+			$attribute_value  = $p->get_attribute( $attribute_name );
+			$result           = $this->evaluate( $attribute_value, end( $namespace_stack ), end( $context_stack ) );
+
+			// Gets the content between the template tags and leaves the cursor in the closer tag.
+			$inner_content = $p->get_content_between_balanced_template_tags();
+
+			// Checks if there is a manual server-side directive processing.
+			$template_end = 'data-wp-each: template end';
+			$p->set_bookmark( $template_end );
+			$p->next_tag();
+			$manual_sdp = $p->get_attribute( 'data-wp-each-child' );
+			$p->seek( $template_end ); // Rewinds to the template closer tag.
+			$p->release_bookmark( $template_end );
+
+			/*
+				* It doesn't process in these situations:
+				* - Manual server-side directive processing.
+				* - Empty or non-array values.
+				* - Associative arrays because those are deserialized as objects in JS.
+				* - Templates that contain top-level texts because those texts can't be
+				*   identified and removed in the client.
+				*/
+			if (
+				$manual_sdp ||
+				empty( $result ) ||
+				! is_array( $result ) ||
+				! array_is_list( $result ) ||
+				! str_starts_with( trim( $inner_content ), '<' ) ||
+				! str_ends_with( trim( $inner_content ), '>' )
+				) {
+				array_pop( $tag_stack );
+				return;
+			}
+
+			// Extracts the namespace from the directive attribute value.
+			$namespace_value         = end( $namespace_stack );
+			list( $namespace_value ) = is_string( $attribute_value ) && ! empty( $attribute_value )
+			? $this->extract_directive_value( $attribute_value, $namespace_value )
+			: array( $namespace_value, null );
```

Comment:
Just some indentation:

```suggestion
				? $this->extract_directive_value( $attribute_value, $namespace_value )
				: array( $namespace_value, null );
```

Reply:
🤦  Fixed in https://github.com/WordPress/wordpress-develop/pull/5953/commits/e0533e555f5b3a76c00c266e383cabd64e63ef5c.